yarn-error.log

```diff
@@ -0,0 +1,1909 @@
+Arguments: 
```

Comment:
Probably shouldn't have this file in the repo. Suggest deleting it and adding to `.gitignore`.

Reply:
ok .. will remove .. thanks @jseppi

Reply:
removed and added to gitignore